app/templates/search/_services_save_search.html

```diff
@@ -4,7 +4,7 @@
   <form action="{{ url_for('direct_award.save_search', framework_family=framework_family) }}">
     <input type="hidden" name="search_query" value="{{ search_query_url }}">
     <div class="button-save-wrapper govuk-grid-row">
-      <div class="save-button govuk-grid-column-one-quarter">
+      <div class="save-button govuk-grid-column-full">
```

Comment:
Do we need both these wrapper divs and their classes? Can we replace them with one div and some govuk-frontend overrides?

Reply:
Good thinking! Doing a little digging, there's quite a bit of unnecessary CSS we can jettison as well:

`.button-save` is the old button class, made redundant by using the new govuk button. 
`.save-summary-text` is no longer used anywhere

The only behaviour we need on the button is to align it to the right. I had a look, and there doesn't seem to be a built in govuk class for that, so I've kept the wrapper to apply `text-align: right;`.

But it'd be even neater to drop the wrapper as well. There are some examples on govuk frontend of using `position: absolute` to align right, but that's more CSS, and it needs a `position: relative` parent anyway, so I _think_ this is the neatest way.

Reply:
Ooh much nicer!

One other suggestion, I've heard that it's good practice to put `app-` in front of CSS classes that are specific to the app rather than a toolkit (I think it's GOV.UK style, not sure where it's documented though).

Anyway we've been trying to add `app-` to CCS styles that we want to keep and leaving it off old styles, so we know what we're happy with.

TL;DR I think we can rename this class to `app-button-save-wrapper`

Reply:
Oh and for extra pedantry we could try naming it using [BEM style](http://getbem.com/naming/), which is the methodology the Design System uses.

Reply:
I've gone with `app-button--right` since I think BEM style modifiers should indicate what they do.